frontend/src/plaquettes/Template.js

```diff
@@ -1,230 +1,87 @@
-import { Container, Graphics, Color } from 'pixi.js';
+import { Container, Graphics } from 'pixi.js';
 import Plaquette from './Plaquette';
 import notification from '../components/notification';
 import Button from '../components/Button';
 
+/**
+ * Template class to create the template for the plaquettes
+ * @class Template
+ * @param {Container} workspace - The workspace container
+ * @param {PIXI.Application} app - The PIXI application
+ * @param {number} x - The x coordinate
+ * @param {number} y - The y coordinate
+ * @returns {void}
+ */
 export default class Template {
-  constructor(selectedQubits, workspace, plaquetteButton, app) {
+  constructor(workspace, app, x, y) {
     // UI Properties
     this.app = app;
     this.container = new Container();
     this.buttonMode = true;
     this.cursor = 'pointer';
     this.mode = 'static';
     this.isDragging = false;
+    this.x = x;
+    this.y = y;
     this.startX = 0;
     this.startY = 0;
     this.plaquette = null;
-    this.plaquetteButton = plaquetteButton;
-    this.templateQubits = selectedQubits || [];
+    this.selectQubitsButton = new Button('Select 3 qubits to make a Plaquette', x, y + 50);
+    this.unselectQubitsButton = new Button('Unselect Qubits', x, y + 100);
+    this.createPlaquetteButton = new Button('Make Plaquette', x, y + 50);
     this.selectedQubits = [];
     this.rectangle = new Graphics();
     this.workspace = workspace;
-    const { x, y } = workspace.mainButtonPosition;
-    this.templateButton = new Button('Step 1: Define a Template', x, y + 50);
-    this.templateButton.on('click', () => {
-      // Create the template
-      this.renderTemplateControlButtons();
-      this.defineTemplateArea();
-      // Clear the selected qubits
-      this.selectedQubits = [];
-      // Hide the button
-      this.templateButton.visible = false;
-
-      // Show a notification to now select qubits within the template to make a plaquette
-    });
-    this.container.addChild(this.templateButton);
-    this.container.addChild(this.plaquetteButton);
     this.container.name = 'template';
-    // this.workspace.addChild(this.container);
+    this.initializeTemplate();
   }
 
-  // Render the template control buttons
-  renderTemplateControlButtons() {
```

Comment:
We can still keep this functionality. That goes for most of the deleted code here

Reply:
do we want to leave it uncommented?

Reply:
Yes. If it's not being invoked at the moment, it should remain uncommented.

Reply:
added it back